### ISSUE TITLE
How to make it work?

### ISSUE THREAD

**Issue by arthurabreu1** [endoplasmic/google-assistant#102]
Hello,

I am new to JS / NODE.js...  sorry in advance for the silly question.
Wondering how I can make this work... I have installed node.js on my PC.
Trying to run executing npm run and the only script there is called test.
when I try npm run test it just have a code to exit... wondering how to make the examples work.

Thanks

**Comment by Aymkdn** [comment]
You have to create a JS script with the provided example : https://github.com/endoplasmic/google-assistant#usage

Make sure to modify with your own information. Then run node with the created script.

**Comment by arthurabreu1** [comment]
thank you, it worked...
I am now trying to use the text-based conversation with my google assistant... auth is working fine apparently but can't seem to get an answer... 
![image](https://user-images.githubusercontent.com/50879554/135772284-7d270667-a125-4b95-8363-d71ae50b905a.png)

**Comment by Aymkdn** [comment]
You may want to try this sample : https://github.com/endoplasmic/google-assistant/blob/master/examples/console-input.js

**Comment by endoplasmic** [comment]
Yah sadly even though you hear the response it doesn't always come with a text string :( Glad you got it working though!

**Comment by arthurabreu1** [comment]
yeah, that's exactly what I noticed. some commands return a text response but others don't. not sure why. but thanks a lot for helping me to get this working.
